app/assets/stylesheets/_sign_in.scss

```diff
@@ -0,0 +1,64 @@
+.border_right {
+  padding-right: 10px;
+  border-right: 1px solid #ccc;
+}
+
+.custom_size {
+  font-size: 70px;
+}
+
+.headers {
+  padding-bottom: 30px;
+}
+
+.slogan {
+  padding-top: 70px;
+  padding-bottom: 15px;
+}
+
+hr {
+  width: 60%;
+  text-align: center;
+}
```

Comment:
Don't overwrite a default global element style. Add a class to it, and specify custom rules in that class.